Library/Homebrew/software_spec.rb

```diff
@@ -122,9 +122,12 @@ def option(name, description = "")
         puts "Symbols are reserved for future use, please pass a string instead"
         name = name.to_s
       end
+      unless String === name
```

Comment:
Is this doing `name.is_a? String`? If so, I'd find that more obvious personally.